### ISSUE TITLE
Interfaces are not supported

### ISSUE THREAD

**Issue by antogyn** [Escape-Technologies/graphman#47]
Hello, since it's asked in the error message and I didn't find an issue already referencing this - running graphman on my schema returns this error:

```
error: Uncaught (in promise) Error: Kind INTERFACE for type <TheInterfaceName> is not supported yet. Please open an issue.
```

Here is an exemple with a public schema:

```
deno run https://deno.land/x/graphman@v1.1.3/src/cli.ts https://swapi-graphql.netlify.app/.netlify/functions/index
```

**Comment by nohehf** [comment]
It's a known limitation, for now, I'll add support for this type as soon as I have some bandwidth to update the repo. Thanks for pointing that out! :)

**Comment by nohehf** [comment]
Fixed in v1.1.4 :)
